MinVer/Logger.cs

```diff
@@ -48,6 +48,9 @@ public void Warn(int code, string message)
         public static void ErrorRepoOrWorkDirDoesNotExist(string repoOrWorkDir) =>
             Error(1002, $"Repository or working directory '{repoOrWorkDir}' does not exist.");
 
+        public static void ErrorInvalidAutoIncrement(string autoIncrement) =>
+            Error(1006, $"Invalid auto increment '{autoIncrement}'. Valid values are {VersionPart.Patch.GetValidValues()}");

```

Comment:
This is a bit of a hack, since it's actually taking one of the enum values in order to call an extension method on it, but the alternative is something like

```C#
VersionPartExtensions.GetValidValues()
```

Given enums are value types, I think this hack is OK.

Reply:
I agree, not pretty but the functionality is actually where it belongs...